tool/gravity/cli/run.go

```diff
@@ -81,7 +76,15 @@ func Run(g *Application) error {
 	if err != nil {
 		return trace.Wrap(err)
 	}
-	return Execute(g, cmd, extraArgs)
+
+	cmdString := fmt.Sprintf("gravity %s", strings.Join(os.Args[1:], " "))
+	cmdString = SanitizeCmd(g, cmd, cmdString)
+
+	LogCLIRunning(cmdString)
+	err = Execute(g, cmd, extraArgs)
+	LogCLICompleted(cmdString, err)
```

Comment:
I would defer this and extended to catch/log panics as well (they sometimes do happen).

```go

logCLIRunning(cmdString)
defer logCLICompleted(cmdString, err)
err = Execute(g, cmd, extraArgs)

```